source/prototype.html.md.erb

```diff
@@ -0,0 +1,142 @@
+---
+title: PROTOTYPE - 1.1.1 Non-text Content (A)
+---
+
+# <%= current_page.data.title %>
+
+<%= warning_text('This is a temporary prototype that is intended to show how the content for the WCAG Primer and the Test Guide might work together when merged.') %>
+
+Current pages for comparison: [WCAG Primer: 1.1.1](sc/1.1.1) and [Test Guide: 1.1.1](https://github.com/alphagov/wcag-primer/wiki/1.1.1)
+
+## [WCAG says](https://www.w3.org/WAI/WCAG22/Understanding/non-text-content):
```

Comment:
I think the link to the criterion might work better being part of the body content rather than being within the heading. 

It seems a little confusing given the heading is itself an anchor point is linked from the sidebar navigation, resulting in the page having two links labelled "WCAG says" that lead to different places.

Reply:
That makes sense, thanks. I've moved the link just below the heading, using the naming convention we currently use in the test guide.